src/scss/main.scss

```diff
@@ -16,3 +16,9 @@
 'base/typography',
 'base/utilities/hide',
 'base/generic';
+
+// Components
+@import
+'components/input',
+'components/label',
+'components/example'
```

Comment:
Currently loading in the example component with the main design system styles. Would be good to bounce ideas on how best to separate out the docs-specific styles from the main design system without introducing too much duplication (could be work that's covered in a later ticket)

Reply:
Currently, the docs-specific styles are written in Tailwind for this reason.

I think we need to decide wether we are happy to:

1. mix and match tailwind with some Sass
2. refactor now (or very soon) to move the tailwind styles into Sass
3. stick to tailwind only

My preference is 3 (I want to move quickly and tailwind helps that) at the moment but happy to consider the other options.

Reply:
I see now. I agree with approach 3 - I'll refactor the example component's markup to be written with Tailwind